### PR TITLE
[BASKET-3492]Apply ADR black + isort

### PR DESCRIPTION
**Pull request by aayush-zen**
- [x] Apply ADR Black + isort to the repo
- [x] Add pre-commit and pyproject config

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+default_language_version:
+  python: python3.6
+repos:
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
+    hooks:
```
Comment (himanijain1) [inline .pre-commit-config.yaml:11]: I think we need args also inside hooks:
ref https://github.com/zendesk/zopim-automation/pull/1308/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R8

File: README.rst [hunk 2]
```diff
@@ -95,3 +95,12 @@ Now, with this having been created, we can go ahead and use it::
...
 .. _MemCacheProtocol: http://twistedmatrix.com/documents/current/api/twisted.protocols.memcache.MemCacheProtocol.html
+
+Python code style
+-----------------
+
+This repository adopts the `black <https://github.com/psf/black>`_ code style and uses `isort <https://github.com/timothycrosley/isort/>`_ to sort all imports. To lint your changes with ``black`` and ``isort``, install `pre-commit <https://pre-commit.com/>`_ and run::
+
+    cd txconnpool
+    pre-commit install
+    pre-commit run --all-files
```
Comment (himanijain1) [inline README.rst:122]: Can add badge for `code-style` and `pre-commit`

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      - id: black
+  - repo: https://github.com/timothycrosley/isort
+    rev: 5.8.0
+    hooks:
+      - id: isort
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.5.0
+    hooks:
+      - id: end-of-file-fixer
```
Comment (himanijain1) [inline .pre-commit-config.yaml:26]: Can we add flake8 pre-commit hook?

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+default_language_version:
+  python: python3.6
```
Comment (aayush-zen) [inline .pre-commit-config.yaml:2]: Should this be `pypy3` ?

**Comment by himanijain1** [comment]
Also add the commits to `.git-blame-ignore-revs` file.

**Comment by aayush-zen** [comment]
> Also add the commits to .git-blame-ignore-revs file.

I guess, It's redundant for such small set of changes

File: setup.py [hunk 5]
```diff
@@ -1,6 +1,6 @@
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
 
-version = '0.1.2'
+version = "0.1.3"
```
Comment (aayush-zen) [inline setup.py]: Should we bump this version or leave as is?

Comment (exvn4697) [reply setup.py]: maybe later when we want to publish new version like in https://github.com/zendesk/txconnpool/commit/1892116782cce13f4ec2b8b60731cbd1b5492260

**Review by exvn4697 — APPROVED** [review]

**Review by oswellchan — APPROVED** [review]

**Review by himanijain1 — APPROVED** [review]

**Pull request merged**